src/test/scala/org/constellation/domain/transaction/TransactionServiceTest.scala

```diff
@@ -295,7 +297,7 @@ class TransactionServiceTest
 
   "pullForConsensusSafe" - {
     // Ignored because of non deterministic output (4999 isn't exual 5000).
-    "should be safe to use concurrently" ignore {
+    "should be safe to use concurrently".ignore {
```

Comment:
It's a minor thing, but I would keep the spaces here and below. Probably Intellij did this refactor.